src/sap.m/src/sap/m/FeedInput.js

```diff
@@ -178,7 +178,7 @@ sap.ui.define([
 
 	FeedInput.prototype.setIconDensityAware = function (iIconDensityAware) {
 		this.setProperty("iconDensityAware", iIconDensityAware, true);
-		var fnClass = sap.ui.require("sap/m/Image");
```

Comment:
I think the idea behind the existing check was not to force loading of ``sap/m/Image`` when it is not needed. That's valid and done in several other places, but it is mandatory to check whether `fnClass` is not null:

```js
if (fnClass && this._getImageControl() instanceof fnClass) {
```

Your proposal also fixes the runtime error but `sap.ui.requireSync` is not a public API (as we don't want to encourage sync loading) and it enforces loading.

If you don't mind, I can adapt your change accordingly before integrating it or you can do it in this PR. Just let me know.

Reply:
And please don't forget to sign the CLA.